tests/endpoints/test_voyages_congestion_breakdown.py

```diff
@@ -42,11 +42,12 @@ def test_search_returns_larger_size(self):
         start = datetime(2019, 1, 1)
         end = datetime(2019, 11, 10)
 
+        # Limit breakdown to 25 to prevent too many ES buckets down-the-line
```

Comment:
I think that the problem is that we are using breakdown_port [here](https://github.com/VorTECHsa/python-sdk/blob/7553d7622df36929af3654b50a1b2ee216712604/vortexasdk/endpoints/voyages_congestion_breakdown.py#L30) and it should be one of these instead `vessel_count,utilisation,cargo_quantity,avg_wait_time,dwt,cubic_capacity,tonne_miles,avg_distance,avg_speed`